spec/unit/shoulda/matchers/active_record/validate_uniqueness_of_matcher_spec.rb

```diff
@@ -450,9 +450,19 @@
 
         expect(record).to validate_uniqueness
       end
+      context 'when the value has no case sensitivity' do
```

Comment:
Maybe "when the value is the same as its swapcased version"?

Also add a line break above this line, per style.